src/Http/Controllers/Api/Geojson/ImageAnnotationsController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Biigle\Modules\Geo\Http\Controllers\Api\Geojson;
+
+use DB;
+use Biigle\Image;
+use GeoJson\Feature\{Feature, FeatureCollection};
+use GeoJson\Geometry\Point;
+use Biigle\Http\Controllers\Api\Controller;
+use League\Flysystem\FileNotFoundException;
+
+class ImageAnnotationsController extends Controller{
```

Comment:
Please use spaces before curly braces. This should be done in the rest of the code, too.
```suggestion
class ImageAnnotationsController extends Controller {
```